index.js

```diff
@@ -875,6 +901,11 @@
     }, of(typeRep, {}));
   }
 
+  //  Function$id :: () -> a -> a
+  function Function$id() {
+    return function(x) { return x; };
```

Comment:
It may be nicer to return the same identify function every time, rather than creating a new identity function over and over again. Currently `S.id(Function) !== S.id(Function)`.